src/cdk/testing/testbed/fake-events/event-objects.ts

```diff
@@ -46,8 +46,15 @@ export function createMouseEvent(
 
   // IE won't set `defaultPrevented` on synthetic events so we need to do it manually.
   event.preventDefault = function() {
-    defineReadonlyEventProperty(event, 'defaultPrevented', true);
-    return originalPreventDefault();
+    /*
+      Calling originalPreventDefault before the if check ensures
+      that non IE apps never call defineReadonlyEventProperty
+    */
+    const result = originalPreventDefault();
+    if (!event.defaultPrevented) {
```

Comment:
Doesn't this mean that it could still fail if IE calls `preventDefault` multiple times?

Reply:
No - If `preventDefault` is called then `defaultPrevented` is true and the property will not be redefined. As you said it would also work if we call it with `configurable: true`. By default `configurable` is false.